Description: Official upstream patch.
 Fix 1. EMBOSS-6.3.1/configure
        EMBOSS-6.3.1/m4/mysql.m4
 .
 21 Jul 2010: Addresses a problem whereby, in some circumstances, inclusion of
              hpdf support prevented inclusion of MySQL support.
              Using the mysql.m4 from this patch may cause a libtool
              error during compilation if you have a version of libtool
              greater than 2.2.6b on your system. If this happens type
              "autoreconf -fi" after adding the file. Note that you
              do not need to use the mysql.m4 file and it is probably
              safer not to; just use the configure file.
 .
 Fix 2. EMBOSS-6.3.1/configure
 .
 6 Aug 2010: This is exactly the same as Fix 1 above, but the associated
             file in the patches directory does not contain the
             mysql.m4 file - it is unnecessary for the operation of the
             EMBOSS distribution. Using this patch will not cause the
             libtool problem described above.
Origin: ftp://emboss.open-bio.org/pub/EMBOSS/fixes/patches/patch-1-2.gz
Forwarded: not-needed
diff -c -N --recursive -a EMBOSS-6.3.1old/configure EMBOSS-6.3.1/configure
*** EMBOSS-6.3.1old/configure	2010-07-19 12:55:36.000000000 +0100
--- EMBOSS-6.3.1/configure	2010-08-06 09:06:53.000000000 +0100
***************
*** 14481,14488 ****
  
      	    EMBCFLAGS=$CFLAGS
  	    EMBLDFLAGS=$LDFLAGS
! 	    CFLAGS=$MYSQL_CFLAGS
! 	    LDFLAGS=$MYSQL_LDFLAGS
  
              cat confdefs.h - <<_ACEOF >conftest.$ac_ext
  /* end confdefs.h.  */
--- 14481,14488 ----
  
      	    EMBCFLAGS=$CFLAGS
  	    EMBLDFLAGS=$LDFLAGS
! 	    CFLAGS="$MYSQL_CFLAGS $EMBCFLAGS"
! 	    LDFLAGS="$MYSQL_LDFLAGS $EMBLDFLAGS"
  
              cat confdefs.h - <<_ACEOF >conftest.$ac_ext
  /* end confdefs.h.  */
